ICT4DNews/app/src/main/kotlin/at/ict4d/ict4dnews/screens/more/MoreScreen.kt

```diff
@@ -0,0 +1,257 @@
+    )
+}
+
+@Composable
+private fun Button(text: String, onClick: () -> Unit) {
+    Button(
+        onClick = onClick,
+        colors = ButtonDefaults.buttonColors(MaterialTheme.colorScheme.primary),
+        shape = RoundedCornerShape(24.dp),
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(horizontal = 16.dp, vertical = 2.dp),
+    ) {
+        Text(
+            text = text, fontWeight = FontWeight.Bold
+        )
+    }
+}
+
+@Composable
+private fun TextAboutDevelopers(text: String) {
+    Text(
+        text = text,
+        modifier = Modifier.padding(horizontal = 16.dp),
+        textAlign = TextAlign.Justify,
+        fontSize = 16.sp,
+    )
+}
+
+@Composable
+private fun DeveloperProfile(
+    developer: ICT4DDeveloper,
+    onClick: () -> Unit
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = Modifier.clickable { onClick.invoke() }
+    ) {
+        Image(
+            painter = painterResource(developer.image),
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            alignment = Alignment.Center,
+            modifier = Modifier.size(72.dp)
+        )
+        Text(
+            text = developer.name,
+            fontWeight = FontWeight.Bold,
+            fontSize = 16.sp,
+            modifier = Modifier.padding(top = 8.dp)
+        )
+        Text(
+            text = developer.role,
+            fontSize = 16.sp,
+        )
+    }
+}
+
+private data class ICT4DDeveloper(
+    val image: Int,
+    val name: String,
+    val role: String,
+    val url: Int
+)
+
+@Composable
+private fun ListOfDevelopers(onOpenDevUrl: (Int) -> Unit) {
+
+    val ict4dDevs = listOf(
+        ICT4DDeveloper(
+            image = R.drawable.paul,
+            name = stringResource(R.string.paul_spiesberger),
+            role = stringResource(R.string.software_developer),
+            url = R.string.url_paul
+        ),
+        ICT4DDeveloper(
+            image = R.drawable.raja,
+            name = stringResource(R.string.raja_saboor_ali),
+            role = stringResource(R.string.software_developer),
+            url = R.string.url_raja
+        ),
+        ICT4DDeveloper(
+            image = R.drawable.noah,
+            name = stringResource(R.string.noah_alorwu),
+            role = stringResource(R.string.software_developer),
+            url = R.string.url_noah
+        ),
+        ICT4DDeveloper(
+            image = R.drawable.job,
+            name = stringResource(R.string.job_guitiche),
+            role = stringResource(R.string.software_developer),
+            url = R.string.url_job
+        ),
+        ICT4DDeveloper(
+            image = R.drawable.chloe,
+            name = stringResource(R.string.chlo_zimmermann),
+            role = stringResource(R.string.designer),
+            url = R.string.url_chloe
+        ),
+    )
+    val shuffledIct4dDevs = ict4dDevs.shuffled()
+
+    DevProfilesRow(
```

Comment:
Instead of creating this layout manually, lets use a [Flow Layout](https://developer.android.com/jetpack/compose/layouts/flow)  instead. This will make the view dynamic and it will look good on small and big screens. Also it will be much easier to maintain and add future contributors :+1:

Reply:
Excellent suggestion! 👍